tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1304,6 +1267,14 @@ private void writeSummaryRow(Appendable output, EnumSet<Choice> choices, Counter
         output.append("</tr>\n");
     }
 
+    private String getLocaleProgressPercent(String localeId, Counter<Choice> problemCounter) {
+        final LocaleCompletionData lcd = new LocaleCompletionData(problemCounter);
+        final int problemCount = lcd.problemCount();
+        final int total = localeBaselineCount.getBaselineProblemCount(CLDRLocale.getInstance(localeId));
```

Comment:
should there be a column for the baseline count also?  since this is HTML it could be in a `title=`

Reply:
I can't really judge the value of adding another column -- I'm just trying to implement what's called for in the ticket for now

Reply:
I agree that the baseline count would be useful, but that can come later; shouldn't interfer with this PR